app/assets/javascripts/questions.js

```diff
@@ -26,7 +26,21 @@ function initEditQuestion() {
   $('#question_text, #question_answers_attributes_0_text').keyup(validateEditQuestionInput);
 
   // track qset user created a question in
-  $('#question_qset_id').change(function() {
+  $('#qset_id_input').change(function() {
     Cookies.set('new_question_qset_id', this.value, { expires: 1000 });
   });
+
+  hasQuestionsIsSelectable();
+
+}
+
+// Closes modal, displays qset name next to Choose Qset button, and populates hidden qset-id input
+function hasQuestionsIsSelectable() {
```

Comment:
I was confused by this function name at first, until I saw it in context. The name appears to be describing a qset, but doesn't describe what the function does. The comment helps, but the code would be easier to read elsewhere if the function name described the purpose/result of the function, instead of the type of qset the function might be attached to. E.g. `selectQsetForNewQuestion()` or something...?